File: support_tools/swigwin-3.0.12/Examples/test-suite/php/extend_template_ns_runme.php
<?php

require "tests.php";
require "extend_template_ns.php";

check::classes(array("extend_template_ns","Foo_One"));
$foo=new Foo_One();
check::equal(2,$foo->test1(2),"test1");
check::equal(3,$foo->test2(3),"test2");

check::done();
?>

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/extend_template_ns_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'extend_template_ns'

include Extend_template_ns

f = Foo_One.new
if f.test1(37) != 37
  raise RuntimeError
end

if f.test2(42) != 42
  raise RuntimeError
end
